benchmarks/MOABB/models/BraindecodeNN.py

```diff
@@ -29,6 +32,7 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
             Input to convolve. 4d tensors are expected.
         """
         # (batch, time_, EEG channel, channel) ->  # (batch, EEG channel, time_, channel)
-        x = torch.transpose(x, 1, 2)
+        x = self.input_layer(x)
         x = self.model(x)
+        x = self.softmax(x)
```

Comment:
It appears that some braindecode models already have the LogSoftmax layer (EEGInception for example), in those cases this will produce a bug. Until the braindecode models are standardized, we should add a flag to the constructor for whether or not to apply the softmax.

Reply:
This is something to discuss the the BrainDecode team

Reply:
hey @Drew-Wagner and @mravanelli!

Since the last code sprint last summer, we have moved to remove the log softmax layer, and in the source version, we have already removed https://github.com/braindecode/braindecode/pull/672

Reply:
@bruAristimunha do you plan to release an updated version on pyPI?